src/main/java/net/fabricmc/loom/build/MixinRefmapHelper.java

```diff
@@ -94,6 +104,15 @@ public static boolean addRefmapName(Project project, Path outputPath) {
 		}
 	}
 
+	private static String getRootPath(SourceSet sourceSet) {
+		File root = Iterables.getFirst(sourceSet.getResources().getSrcDirs(), null);
```

Comment:
Hmm, it could also be nice to search for the mixin file in all resource directories of the source set (this `Iterables.getFirst` can cause unexpected errors when splitting source sets into many directories)